frontend/src/lib/k8s/endpoints.ts

```diff
@@ -44,7 +44,7 @@ class Endpoints extends makeKubeObject<KubeEndpoint>('endpoint') {
   }
 
   getAddressesText() {
-    this.getAddresses().join(', ');
+    return this.getAddresses().join(', ');
```

Comment:
This should go in a different commit, maybe with the Co-Authored-By: pointing to @Src0p  (we'll need this user's name and email, I will ask on Slack).

Reply:
removed the return keyword